io/net/virt-net/lpm.py

```diff
@@ -55,7 +55,7 @@ def setUp(self):
         self.hmc_user = self.params.get("hmc_username", default='hscroot')
         self.hmc_pwd = self.params.get("hmc_pwd", '*', default='********')
         self.options = self.params.get("options", default='')
-
+        self.param = self.params.get("param", default='')
```

Comment:
Is this not used ?

Reply:
yes it is used

Reply:
@abdhaleegit This variable is used, but not from yaml.